packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -57,59 +42,82 @@ module.exports = function handleTransaction({
+The `IdxStatus` field indicates `IdxStatus.SUCCESS`, retrieves the token from the response, and processes the authenticated user in the app. The SDK sample application saves the tokens to storage in the `handleTransaction.js` file and redirects the user back to the home page.
 
 ```JavaScript
 case IdxStatus.SUCCESS:
-      // Save tokens to storage (req.session)
-      authClient.tokenManager.setTokens(tokens);
-      // Redirect back to home page
-      res.redirect('/');
-      return;
+  // Save tokens to storage (req.session)
+  authClient.tokenManager.setTokens(tokens);
+  // Redirect back to home page
+  res.redirect('/');
+  return;
 
 ...
 
 ```
 
 #### Other authentication statuses
 
-You need to handle other returned `IdxStatus` cases if the user didn't sign in successfully. For example, in the SDK application's `handleTransactions.js` file:
+If the user sign-in didn't result in a success status, then you need to handle the returned `IdxStatus` cases. For example, in the SDK application's `handleTransactions.js` file:
 
 ```JavaScript
- switch (status) {
-    case IdxStatus.PENDING:
-      // Proceed to next step
-      try {
-        if (!proceed({ req, res, nextStep })) {
-          next(new Error(`
-            The current flow cannot support the policy configuration in your org.
-          `));
-        }
-      } catch (err) {
-        next(err);
+switch (status) {
+  case IdxStatus.PENDING:
+    // Proceed to next step
+    try {
+      if (!proceed({ req, res, nextStep })) {
+        next(new Error(`
+          The current flow cannot support the policy configuration in your org.
+        `));
       }
-      return;
-    case IdxStatus.SUCCESS:
-      // Save tokens to storage (req.session)
-      authClient.tokenManager.setTokens(tokens);
-      // Redirect back to home page
-      res.redirect('/');
-      return;
-    case IdxStatus.FAILURE:
-      authClient.transactionManager.clear();
-      next(error);
-      return;
-    case IdxStatus.TERMINAL:
-      redirect({ req, res, path: '/terminal' });
-      return;
-    case IdxStatus.CANCELED:
-      res.redirect('/');
-      return;
-  }
+    } catch (err) {
+      next(err);
+    }
+    return;
+  case IdxStatus.SUCCESS:
+    // Save tokens to storage (req.session)
+    authClient.tokenManager.setTokens(tokens);
+    // Redirect back to home page
+    res.redirect('/');
+    return;
+  case IdxStatus.FAILURE:
+    authClient.transactionManager.clear();
+    next(error);
+    return;
+  case IdxStatus.TERMINAL:
+    redirect({ req, res, path: '/terminal' });
+    return;
+  case IdxStatus.CANCELED:
+    res.redirect('/');
+    return;
+}
 
 ```
 
-### 4 (Optional): Get the user profile information
+### 3 (Optional): Get the user profile information
+
+After the user is authenticated, you can get the user profile information by making a request to the Okta OpenID Connect authorization server.
```

Comment:
The optional step is back of the procedure and should appear as part of the sequence.